Lock PC prank payload

fratervi opened the pull request:
I created the OS detection extension based on the win93 prank and modified the lock pc prank to utilize it..

thehappydinoa commented:
This extension was implemented in #277. But the prank looks good. Just update to use GET TARGET_OS

sebkinne commented on payloads/extensions/get_os.sh:
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```
The same functionality was added to the GET extension. If you remove this, we are good to merge

sundhaug92 commented:
Also, change the readme.md and payload (GET_OS OS to GET TARGET_OS for example)

fratervi commented:
Thanks, I am attempting to implement those changes.  I'm having some difficulty getting the "GET TARGET_OS" command to return a value.  I imagine I am missing something obvious, but I will sort it out.

sebkinne commented on payloads/library/prank/lockpc/payload.txt:
```diff
@@ -0,0 +1,68 @@
+#		     it can be run on any(mostly any) OS.
+#
+#options
+mount -o /dev/nandf /root/udisk
```
Please don't do this - nandf is already mounted to /root/udisk. If for some reason it wasn't, please use the following command: `udisk mount`

sebkinne commented on payloads/library/prank/lockpc/payload.txt:
```diff
@@ -0,0 +1,68 @@
+esac
+
+LED CLEANUP
+umount /root/udisk
```
Once your payload is done running it will automatically unmount udisk. If you wish to do so manually, please run `udisk unmount`

sebkinne commented:
@fratervi Let me know if there is an issue with the extension :)

fratervi commented:
@sebkinne  I had been having trouble with the nandf mounting during payload execution.  But that turned out to be some corruption on the file system.  After a quick round of fsck ,that issue went away, so I removed those lines from the payload.  However, I was still having trouble getting Get to return the OS value to me.  I was playing with the get.sh and changed OSFound to TARGET_OS and it returned a value correctly.  Am I misunderstanding the implementation?  the command used in the payload was "GET TARGET_OS"

Thanks for the rapid feedback!

sebkinne commented:
@fratervi you seem to be right, the change made by @thehappydinoa to the GET extension requires TARGET_IP to already be exported. I have pushed a fix, which should resolve your issues.

thehappydinoa commented:
@fratervi @sebkinne Thank you for fixing my mistake.
